opm/parser/eclipse/EclipseState/Tables/FoamadsTable.hpp

```diff
@@ -0,0 +1,37 @@
+/*
+  Copyright (C) 2014 by Andreas Lauser
```

Comment:
I guess that copyright statement is not entirely correct?

Reply:
added copyright statement